code/modules/jobxp/JobXPRewards.dm

```diff
@@ -244,6 +244,21 @@ mob/verb/checkrewards()
 	required_levels = list("Botanist"=10)
 	path_to_spawn = /obj/item/reagent_containers/glass/wateringcan/rainbow
 
+/datum/jobXpReward/botanist/jumpsuit
+	name = "Senior Botanist Jumpsuit"
+	desc = "An old jumpsuit with an earthy smell to it."
+	required_levels = list("Botanist"=15)
+	icon_state = "?"
+	claimable = 1
+	claimPerRound = 1
+
+	activate(var/client/C)
+		boutput(C, "The jumpsuit pops into existance!")
```

Comment:
```suggestion
		boutput(C, "<span class='hint'>The jumpsuit pops into existance!</span>")
```
Should be styled, otherwise it'll look weird vs other chat text